README.md

```diff
@@ -43,18 +43,42 @@ Install:
 To route mavlink packets from master `ttyS1` to 2 other UDP endpoints, do as
 following:
 
-    $ mavlink-routerd -b 1500000 -e 192.168.7.1:14550 -e 127.0.0.1:14550 /dev/ttyS1
+    $ mavlink-routerd -e 192.168.7.1:14550 -e 127.0.0.1:14550 /dev/ttyS1:1500000
 
-The `-b` switch above is used to set the UART baudrate. See more options with
-`mavlink-routerd --help`
+The `1500000` after colon above on `/dev/ttyS1:1500000` is used to set the
+UART baudrate. See more options with `mavlink-routerd --help`
 
 It's also possible to route mavlinks packets from any interface using:
 
-    $ mavlink-routerd -b 1500000 -e 192.168.7.1:14550 -e 127.0.0.1:14550  0.0.0.0:24550
+    $ mavlink-routerd -e 192.168.7.1:14550 -e 127.0.0.1:14550  0.0.0.0:24550
 
 mavlink-router also listens, by default, port 5760 for TCP connections. Any
 connection there will also receive routed packets.
 
+### INI file ###
+
+It's also possible to use an .ini file to set options for mavlink-routerd.
+By default, mavlink-routerd looks for a file named `mavlink-routerd.ini`
+on its current directory. File location can be overriden via
+`MAVLINK_ROUTER_INI_FILE` environment variable, or via `-i` switch when running
+mavlink-routerd.
+An example of ini file would be:
+
+```
+[General]
+tcp=5790
+report-stats=false
+
+[Master-Endpoints]
+UART=/dev/tty0:52000
+UDP=0.0.0.0:10000
+UDP=0.0.0.0:10001
+
+[Endpoints]
+UDP=127.0.0.1:11000
+UDP=127.0.0.1:11001
+```
+
```

Comment:
I was thinking about a key that would be generic for any endpoint and still allow custom options. But I think it's ok the way you wrote it